app/common/lib/bookmarkUtil.js

```diff
@@ -65,8 +65,18 @@ const getDNDBookmarkData = (state, bookmarkKey) => {
   return data.get('draggingOverKey') === bookmarkKey ? data : Immutable.Map()
 }
 
+let oldSites
+let lastValue
+let lastWidth
 const getToolbarBookmarks = (state) => {
   const sites = state.get('sites', Immutable.List())
+  if (sites === oldSites &&
+      lastWidth === window.innerWidth &&
```

Comment:
good catch, updated PR to only do it once for now, which makes it the same number as before (once).